src/utils.ts

```diff
@@ -353,10 +354,93 @@ export function getRuntimeConfig(config: Partial<IUserConfig> = {}): IRuntimeCon
         }
     }
 
+    // Resolve the paths with the webRoot set by the user
+    const resolvedOverrides: IStringDictionary<string> = {};
+    for (const pattern in sourceMapPathOverrides) {
+        if (sourceMapPathOverrides.hasOwnProperty(pattern)) {
+            const replacePattern = replaceWebRootInSourceMapPathOverridesEntry(webRoot, pattern);
+            const replacePatternValue = replaceWebRootInSourceMapPathOverridesEntry(
+                webRoot, sourceMapPathOverrides[pattern]);
+
+            resolvedOverrides[replacePattern] = replacePatternValue;
+        }
+    }
+
     return {
         pathMapping,
-        sourceMapPathOverrides,
+        sourceMapPathOverrides: resolvedOverrides,
         sourceMaps,
         webRoot,
     };
 }
+
+/**
+ * Find '${webRoot}' in a string and replace it with the specified value only if it is at the start.
+ * @param webRoot The value to use for replacement.
+ * @param entry The path containing the '${webRoot}' string that we will replace.
+ */
+export function replaceWebRootInSourceMapPathOverridesEntry(webRoot: string, entry: string) {
+    if (webRoot) {
+        const webRootIndex = entry.indexOf("${webRoot}");
+        if (webRootIndex === 0) {
+            return entry.replace("${webRoot}", webRoot);
+        }
+    }
+    return entry;
+}
+
+/**
+ * Walk through the list of mappings and find one that matches the sourcePath.
+ * Once a match is found, replace the pattern in the value side of the mapping with
+ * the rest of the path.
+ * @param sourcePath The source path to convert
+ * @param pathMapping The list of mappings from source map to authored file path
+ */
+export function applyPathMapping(

```

Comment:
I'm surprised there isn't any utilities to do pattern matching/replacing for paths already. But this looks like it should work :)

Reply:
I couldn't find any. This is based on the mechanism used by https://github.com/microsoft/vscode-chrome-debug-core/ for sourcemaps.